Makefile.am

```diff
@@ -145,19 +145,32 @@ pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = libvmi.pc
 
 AM_CPPFLAGS = -I$(top_srcdir)/libvmi
-AM_CPPFLAGS += $(ZLIB_CFLAGS)
+if WITH_ZLIB
```

Comment:
No need to make these conditional here, if zlib is not found the flags will just be empty